arch-chroot.in

```diff
@@ -4,11 +4,15 @@ shopt -s extglob
 
 m4_include(common)
 
+setup=chroot_setup
+unshare="$root_unshare"
+
 usage() {
   cat <<EOF
 usage: ${0##*/} chroot-dir [command]
 
     -h                  Print this help message
+    -N                  Run in uNshare mode as a regular user
```

Comment:
I'm also wondering if instead of "unshare mode" it would be better to all it "Run in rootless mode" or something?

Reply:
>Sigstore aims to be the Let’s Encrypt of Code Signing, which is a perfect metaphor to the missing building blocks required for open source supply chain integrity.

I think using '-N' is fine :)

>The reason why I chose this naming is because there several ways to become root (be root already, sudo, unshare, fakeroot, proot, etc.). So I think "unshare" mode is more clear because it specifies the method we use to become root.

Hmm, It's probably fine :) Thanks for the work!